docs/guides/server/hostname.adoc

```diff
@@ -40,7 +40,7 @@ Most of the time, it should be enough to set the `hostname` option in order to c
 When using the `hostname` option the server is going to resolve the HTTP scheme, port, and path, automatically so that:
 
 * `https` scheme is used unless you set `hostname-strict-https=false`
-* Use the standard HTTP ports (e.g.: `80` and `443`) if a `proxy` is set or use the port you set to the `hostname-port` option
+* if `proxy` option is set, the proxy is expected to use the default  ports (e.g.: 80 and 443), in case the proxy uses a different port, it needs to be specified via the `hostname-port` configuration option
```

Comment:
Maybe we should somewhere here also explicitly mention the difference between proxy ports (hostname options) and actual Keycloak ports (http(s)-port options). It's covered by the newly added example but maybe it wouldn't hurt to also mention here?

Reply:
I would maybe add at the very end of the section (bellow the bullet points, on the line no. 52) the following note:
```
[NOTE]
====
By `hostname` and `proxy` configuration options you affect only the links with static resources, redirect URIs, well-known OIDC end-points, etc. 
In other words by setting these you do not change the actual Keycloak server behaviour. 
In order to change the server behaviour (e.g. `http-host`, `https-port`, ...), you need to use the `http/tls` configuration options. For more details, see <@links.server id="enabletls"/> and <@links.server id="all-config"/>.
====
```
Wdyt?

Reply:
Yeah, that placement works for me. I'd just rephrase a bit maybe as we are changing KC behaviour in any case (even with hostname options) just not the ports KC listens on.

Reply:
Addressed in the last commit, PTAL again.